packages/hub/services/serializers/push-notification-registration-serializer.ts

```diff
@@ -1,16 +1,16 @@
-import { PushNotificationRegistration } from '../../routes/push_notification_registrations';
 import { JSONAPIDocument } from '../../utils/jsonapi-document';
+import { push_notification_registrations } from '@prisma/client';
 
 export default class PushNotificationRegistrationSerializer {
-  serialize(model: PushNotificationRegistration): JSONAPIDocument {
+  serialize(model: push_notification_registrations): JSONAPIDocument {
```

Comment:
It is confusing that a singular model is typed as a plural thing. Not sure if that is just a fact of life we have to deal with using Prisma or if there is an alternative.

Reply:
yes, this reflects the larger problem of Prisma’s opinion of database naming conventions not matching our conventions. I’ve made [4256](https://linear.app/cardstack/issue/CS-4256/look-at-automated-prisma-schema-transformation-to-convert-between) to track addressing this. It’s what I’ll focus on next because it will affect all future conversions.